playground/nuxt.config.ts

```diff
@@ -1,7 +1,7 @@
 export default defineNuxtConfig({
-  modules: ['../src/module'],
+  modules: ['../src/module', '@nuxtjs/tailwindcss'],
   stripe: {
-    publishableKey: 'pk_test_123',
-    apiKey: 'sk_test_123'
+    publishableKey: process.env.STRIPE_PKEY,
+    apiKey: process.env.STRIPE_SKEY
```

Comment:
maybe add a `.env.example`

Reply:
I was thinking about it. But it would mean passing the sk key. Even if its a dev mode . I am a bit scared ^^. Will see if we can maybe passe a restricted key here

Reply:
maybe a dedicated stripe's account OR maybe stripe provide test key for demo we could re-use. Will take a look

Reply:
I mean the key I am using is already a demo key here. but even with a demo key people can mess up with the env. Will look if we can restrict to just certains things a key

Reply:
these one from stripe doc https://stripe.com/docs/keys
```
apikey: sk_test_Hrs6SAopgFPF0bZXSN3f6ELN
publishableKey: pk_test_A7jK4iCYHL045qgjjfzAfPxu
```